addons/main/stringtable.xml

```diff
@@ -1051,10 +1059,12 @@
         <Key ID="STR_dui_radar_leading_zeroes">
             <English>Leading Zeroes</English>
             <German>Führende Nullen</German>
+            <Russian>Нули в азимутах</Russian>
         </Key>
         <Key ID="STR_dui_radar_leading_zeroesr_desc">
             <English>The bearing number will be filled with leading zeroes</English>
             <German>Die Richtungsanzeige wird mit führenden Nullen aufgefüllt</German>
+            <Russian>The bearing number will be filled with leading zeroes</Russian>
```

Comment:
Forgotten translations?

Reply:
no) I just don’t really understand how to say this in Russian))

Reply:
That is fine, I would prefer it to be left untranslated then, Arma will take the top most string entry then, which is identical in this case.